sdk/src/Services/MobileAnalytics/Custom/_unity/BackgroundRunner.unity.cs

```diff
@@ -104,6 +104,10 @@ private void DoWork()
                     }
                     Thread.Sleep(BackgroundSubmissionWaitTime * 1000);
                 }
+                catch(ThreadAbortException)
+                {
+                  // handle thread aports more greacefully in unity
```

Comment:
@soapyigu as mentioned in previous comment, it we log an error then it gets logged in the Unity editor as such:
<img width="696" alt="screen shot 2017-04-18 at 5 32 04 pm" src="https://cloud.githubusercontent.com/assets/4657384/25233038/70317246-25ab-11e7-885d-d369b8ad7acb.png">

This is undesirable, and it also gets reported to Unity's Performance Reporting as an exception, also undesirable.